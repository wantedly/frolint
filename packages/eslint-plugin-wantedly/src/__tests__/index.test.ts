import * as plugin from "..";

it("should match snapshot", () => {
  expect(plugin).toMatchSnapshot();
});
