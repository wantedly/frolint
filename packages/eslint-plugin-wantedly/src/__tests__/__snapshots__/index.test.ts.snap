// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
{
  "rules": {
    "graphql-operation-name": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/graphql-operation-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "graphql-pascal-case-type-name": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/graphql-pascal-case-type-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-camel-case-field-name": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-camel-case-field-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-enum-values-description": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-enum-values-description.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-field-description": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-field-description.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-pascal-case-type-name": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-pascal-case-type-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-type-description": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-type-description.md",
        },
        "type": "suggestion",
      },
    },
    "nexus-upper-case-enum-members": {
      "create": [Function],
      "meta": {
        "docs": {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-upper-case-enum-members.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
  },
}
`;
